users/config.rst

```diff
@@ -206,9 +262,15 @@ Folder Element
 
     <folder id="default" label="Default Folder" path="/Users/jb/Sync/" type="sendreceive" rescanIntervalS="3600" fsWatcherEnabled="true" fsWatcherDelayS="10" ignorePerms="false" autoNormalize="true">
         <filesystemType>basic</filesystemType>
-        <device id="3LT2GA5-CQI4XJM-WTZ264P-MLOGMHL-MCRLDNT-MZV4RD3-KA745CL-OGAERQZ"></device>
+        <device id="S7UKX27-GI7ZTXS-GC6RKUA-7AJGZ44-C6NAYEB-HSKTJQK-KJHU2NO-CWV7EQW" introducedBy="">
```

Comment:
Is there any particular reason for changing the existing device IDs (and API keys too, see below)?

Reply:
Yes.  I don't have access to the device certificate originally used to generate the examples.  So it generated a new certificate locally and that one's device ID always got added as a "new" (local) device everywhere.  To avoid fussing around each time I regenerate the XML, I just decided to switch over to the new ID.

Nobody will care I suppose, it's just an example for the encoded format.